wordpress-mobile/gutenberg-mobile
# Update version to 1.28.2

Update the version to 1.28.2

This version disables the Pullquote block for Android.

GB PR: https://github.com/WordPress/gutenberg/pull/22666

To test:
 - Start the demo app in iOS.
 - Check that the pullquote block still shows.
 - Start the demo app in Android
 - Check that the pullquote blocks is not present.

PR submission checklist:
